addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/__tests__/ScEnvironmentCard.test.js

```diff
@@ -86,10 +87,27 @@ describe('ScEnvironmentCard', () => {
     const spyOnConfigsStore = jest.spyOn(component, 'getEnvTypeConfigsStore');
 
     // OPERATE
-    component.getConfiguration(scEnvironment.envTypeConfigId);
+    const config = component.getConfiguration(scEnvironment.envTypeConfigId);
+    console.log(config);
 
     // CHECK
     expect(spyOnConfigsStore).toHaveBeenCalled();
     expect(envTypesStore.getEnvTypeConfig).toHaveBeenCalledWith(scEnvironment.envTypeConfigId);
+    expect(config.name).toBeDefined();
+    expect(config.instanceType).toBeDefined();
+  });
+
+  it('should get undefined configuration', async () => {
+    // BUILD
+    const spyOnConfigsStore = jest.spyOn(component, 'getEnvTypeConfigsStore');
+
+    // OPERATE
+    const config = component.getConfiguration(1);
```

Comment:
The number 1 does not mean anything by itself. Consider changing this with a string like `existingConfigId`.

Reply:
I like this idea! Changing it